omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/ModuleActionResource1_8.java

```diff
@@ -243,11 +246,9 @@ private void startModules(Collection<Module> modules, ServletContext servletCont
 	private void findAndThrowStartupErrors(Collection<Module> modules) {
 		List<Exception> errors = new ArrayList<Exception>();
 		for (Module module : modules) {
-			if (moduleFactoryWrapper.isModuleStopped(module)) {
+			if (moduleFactoryWrapper.isModuleStopped(module) && module.getStartupErrorMessage() != null) {
 				//module actions are executed in other thread, so we need to explicitly check and throw them
-				if (module.getStartupErrorMessage() != null) {
 					errors.add(new ModuleException(module.getStartupErrorMessage()));
```

Comment:
```suggestion
				errors.add(new ModuleException(module.getStartupErrorMessage()));
```
After removing one level of if, doesn't the entire inner block have to be moved to the left? Here and everywhere else where "if" has been removed.

Reply:
I made some changes , waiting for your feedback.